packages/cli/configuration/src/generators-yml/convertGeneratorsConfiguration.ts

```diff
@@ -240,55 +240,87 @@ async function parseAPIConfigurationToApiLocations(
     return apiDefinitions;
 }
 
+async function parseApiConfigurationV2Schema(
+    apiConfiguration: APIConfigurationV2Schema,
+    rawConfiguration: GeneratorsConfigurationSchema
+): Promise<APIDefinition> {
```

Comment:
nit would be good to use an object as the input here `{ apiConfiguration, rawConfiguration }: { apiConfiguration: APIConfigurationV2Schema`